### PR TITLE
removed mock pdf for digitalcard

### PR DESCRIPTION
**Pull request by dhanendra06**

### CODE REVIEW

File: admin/admin-service/src/main/java/io/mosip/admin/controller/ApplicantDetailsController.java [hunk 1]
```diff
@@ -11,6 +11,7 @@
 import org.springframework.core.io.InputStreamResource;
 import org.springframework.http.MediaType;
 import org.springframework.http.ResponseEntity;
+import org.springframework.security.access.prepost.PreAuthorize;
```
Comment (vishwa-vyom) [inline admin/admin-service/src/main/java/io/mosip/admin/controller/ApplicantDetailsController.java:14]: Is this getting used in this class ?

File: admin/admin-service/src/main/java/io/mosip/admin/util/RestClient.java [hunk 2]
```diff
@@ -189,14 +186,40 @@ public <T> T getForObject(String url,
...
+     * Gets the Object.
+     *
+     * @param <T>             the generic type
+     * @param url         the url
+     * @param responseType    the response type
+     * @return the api
+     * @throws Exception
+     */
+    @SuppressWarnings("unchecked")
+    public <T> T getForApi(String url,
```
Comment (vishwa-vyom) [inline admin/admin-service/src/main/java/io/mosip/admin/util/RestClient.java:207]: why do we need this extra method, if get for object takes the same arguments ?

**Review by vishwa-vyom — CHANGES_REQUESTED** [review]

File: admin/admin-service/src/main/java/io/mosip/admin/util/RestClient.java (hunk 2 above)
Comment (dhanendra06) [reply admin/admin-service/src/main/java/io/mosip/admin/util/RestClient.java:207]: we need token here

Comment (dhanendra06) [reply admin/admin-service/src/main/java/io/mosip/admin/util/RestClient.java:207]: and its commonly used method so i have created another one

File: admin/admin-service/src/main/java/io/mosip/admin/controller/ApplicantDetailsController.java (hunk 1 above)
Comment (dhanendra06) [reply admin/admin-service/src/main/java/io/mosip/admin/controller/ApplicantDetailsController.java]: removed unused import

Comment (vishwa-vyom) [reply admin/admin-service/src/main/java/io/mosip/admin/controller/ApplicantDetailsController.java]: resolved

File: admin/admin-service/src/main/java/io/mosip/admin/util/RestClient.java (hunk 2 above)
Comment (vishwa-vyom) [reply admin/admin-service/src/main/java/io/mosip/admin/util/RestClient.java:207]: I am not ok with the name, but for now it is fine

**Review by vishwa-vyom — APPROVED** [review]

**Pull request merged**